### PR TITLE
2594 add market fee sharing

### LINKED ISSUE #2594: [1][dot5enko] Add support for market fee sharing
**Is your feature request related to a problem? Please describe.**

`extensions` field of `asset_options` has a new type in bitshares-core 3.0.0: https://github.com/bitshares/bitshares-core/blob/3.0.0/libraries/chain/include/graphene/chain/protocol/asset_ops.hpp#L85, we need to
- update serialization https://github.com/bitshares/bitsharesjs/issues/45
- update UI to add support for the new extensions
  - [ ] "asset info" page
  - [ ] "create asset" page
  - [ ] "update asset" page
  - [ ] maybe transaction confirmation page
  - [ ] maybe block explorer page
  - [ ] maybe the "create order" panel in market page

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by dot5enko**
<h2>General</h2>

Closes #2594

### CODE REVIEW

**Comment by sschiessl-bcp** [comment]
1. Setting Market fee sharing whitelist and percentage should only be possible if enable market fee flag is on (similar to the inputs "MARKET FEE (%)" in the tab "Flags") (if flag is off, show hint in Whitelist section)

2. Please make the snowflake in setting whitelist smaller (size of line)

3. Reward percent in Flags section
image
  - Header "Reward percent" is unnecessary, remove
  - REWARD PERCENT (%) should be in align with "MARKET FEE (%)" and only be enabled if market fee is set

4. Please add indent such that the three values belonging to "Enable market fee:" also appear as if they belong under it (left indent should be more that "Enable market fee:" and right align should be ending with the toggle)

**Comment by dot5enko** [comment]
image

i've put switch right into heading, removing extra line below. i think its a good idea, isnt it ?
Not sure i have understood 2nd step, can you explain it more?

**Comment by dot5enko** [comment]
@sschiessl-bcp done

File: app/assets/stylesheets/components/_wallet.scss [hunk 1]
```diff
@@ -179,7 +179,8 @@
     padding-bottom: 1em;
 }
 
-.wallet-create, .create-account-wrapper {
+.wallet-create,
```
Comment (sschiessl-bcp) [inline app/assets/stylesheets/components/_wallet.scss]: Please double check all files when committing, this is a reformatting only commit

**Comment by sschiessl-bcp** [comment]
I think the reformatting commits come from merging instead of rebasing.

**Comment by dot5enko** [comment]
yeah, my own changes do not include reformating

**Pull request merged**